src/renderer/components/settings/field/password.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    {{ field.display || field.name }}
+    <button @click="editing = !editing"></button>
+    <div v-if="editing">
+        <label>
+            <input
+                type="password"
+                :name="field.name"
+                :minLength="field.minLength"
+                :maxLength="field.maxLength"
+                :value="value"
+            />
+        </label>
+        <button @click="$emit('value-change', hashedJson(value))"></button>
+    </div>
+</template>
+
+<script>
+  import crypto from 'crypto'
+  
+  const SALT_LENGTH = 6
+
+  function hashedJson (password) {
+    let salt = crypto.randomBytes(Math.ceil(SALT_LENGTH/2))
+            .toString('base64')
+            .slice(0, SALT_LENGTH)
+    let hashedPassword = crypto.createHash('sha256')

```

Comment:
You used let for a variable which doesn't change it value. Eslint is failing the build for it. see [AppVeyor (Our CI)](https://ci.appveyor.com/project/2roy999/launcher-8a1fe/build/1.3.9-41)